ChangeLog

```diff
@@ -23,6 +23,16 @@ Release Date: TBA
 
   Closes #926
 
+* Fix property inference in class contexts for properties defined on the metaclass
+
+  Closes #940
+
+* Update enum brain to fix definition of __members__ for subclass-defined Enums
+
+  Closes PyCQA/pylint#2626
```

Comment:
https://github.com/PyCQA/pylint/issues/2626 doesn't seem to be fixed (at least not the initial example). Am I missing something?

Reply:
You're right, I was looking at the comment in https://github.com/PyCQA/pylint/issues/2626#issuecomment-627442384